README.rst

```diff
@@ -24,14 +24,32 @@ Or, on Linux Mint, run::
   virtualenv py2 --system-site-packages
   source py2/bin/activate
 
+To setup a python 3.5 virtual environment if your default python is 2.7, first
+install python 3.5 via your package manager if needed, and then run::
+
+  virtualenv -p python3.5 ~/py3
+  source ~/py3/bin/activate
+
 To install the dependencies for the basic tools: ``pip install -r requirements.txt``
 
-To install the dependencies for the console GUI, first run ``make deps`` to install the systemwide deps and then ``pip install -r requirements_gui.txt pyside`` for the python deps.
+To install the dependencies for the console GUI, run ``make deps``. Besides
+system packages, this also installs python dependencies into the current
+(virtual) environment, and includes the deps for the aforementioned basic tools.
 
 Finally, ``pip install -e .`` to set up a dev install in the local dev environment.
 
 To run the installed console from the current env, use ``python -m piksi_tools.console.console``
 
+Python version support
+~~~~~~~~~~~~~~~~~~~~~~
+
+The most important command line tools - bootload_v3.py, fileio.py,
+serial_link.py, and settings.py - support Python 2.7, 3.5, and 3.7.
+Console GUI under Linux supports 2.7 and 3.5. Note that at this stage Python 3
+support is still somewhat experimental.
+
+MacOS and Windows builds are still in Python 2.7, and Python 3 support for
```

Comment:
There have been no windows tests run under appveyor before or now, so not necessary here at this point either IMO. A good idea to add at some point in the future. I'll try it for mac, and if it works easily, I'll add, otherwise to be done in the follow-up mac task https://swift-nav.atlassian.net/browse/EX-540.

Reply:
Sounds good, thanks.